tests/test_sphere.py

```diff
@@ -79,7 +79,7 @@ def test_invalid_area_setter(area):
 @given(floats(0.1, 1000))
 def test_iq(r):
     sphere = Sphere(r)
-    assert sphere.iq == 1
+    assert sphere.iq == approx(1)
```

Comment:
This is defined as identically 1.
```suggestion
    assert sphere.iq == 1
```